File: tests/fixtures/sample_code/good_code.java
// good_code.java — clean Java that should produce zero complexity/nesting findings

public class GoodClass {

    /**
     * Adds two integers.
     * @param a first number
     * @param b second number
     * @return sum
     */
    public int add(int a, int b) {
        return a + b;
    }

    /**
     * Returns the larger of two integers.
     * @param x first value
     * @param y second value
     * @return the maximum value
     */
    public int max(int x, int y) {
        if (x > y) {
            return x;
        }
        return y;
    }
}

File: tests/fixtures/sample_code/good_code.js
// good_code.js — clean JS that should produce zero complexity/nesting findings

/**
 * Adds two numbers.
 * @param {number} a
 * @param {number} b
 * @returns {number}
 */
function add(a, b) {
    return a + b;
}

/**
 * Returns the larger of two numbers.
 * @param {number} x
 * @param {number} y
 * @returns {number}
 */
function max(x, y) {
    if (x > y) {
        return x;
    }
    return y;
}
